dataeng/resources/snowflake-collect-queue-depth.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/analytics-tools/snowflake
+make requirements
+
+python collect_queue_depth.py \
+    --key_path $KEY_PATH \
```

Comment:
On the google hangout we were talking about changing this to:
```suggestion
    --key_path $WORKSPACE/analytics-secure/$KEY_PATH \
```

Reply:
but to be honest I would rather it just be completely literal:
```suggestion
    --key_path $WORKSPACE/analytics-secure/snowflake/rsa_key_snowflake_task_automation_user.p8
```

the path to the secret is not itself a secret.

Reply:
Likewise, the all of these things are not secret keys and therefore can be moved into the job DSL file:

* PASSPHRASE_PATH
* SNOWFLAKE_USER
* SNOWFLAKE_ACCOUNT


On a related note, ANALYTICS_TOOLS_URL and ANALYTICS_TOOLS_BRANCH definitions could be moved into common.yaml: https://github.com/edx-ops/analytics-secure/blob/master/job-configs/common.yaml

after all those changes, your job config file would almost be completely nullified!

Reply:
Ok, I can make that change if it's preferred for readability, but that's the main reason, correct?

Reply:
readability is the main reason.